### PR TITLE
Demisto mock: get params, command and arguments from environment variables

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: https://jira-hq.paloaltonetworks.local/browse/CIAC-6919

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/demistomock/demistomock.py [hunk 1]
```diff
@@ -455,6 +467,10 @@ def command():
       str: Integrations command name
 
     """
+    if Path(ARGS_COMMAND_PATH).exists():
+        with Path(ARGS_COMMAND_PATH).open() as f:
```
Comment (dorschw) [inline Tests/demistomock/demistomock.py:471]:
```suggestion
    if (path := Path(ARGS_COMMAND_PATH)).exists():
        with path.open() as f:
```

File: Tests/demistomock/demistomock.py [hunk 2]
```diff
@@ -444,6 +451,11 @@ def args():
       dict: Arguments object
 
     """
+    if Path(ARGS_COMMAND_PATH).exists():
+        with Path(ARGS_COMMAND_PATH).open() as f:
```
Comment (dorschw) [inline Tests/demistomock/demistomock.py:455]:
```suggestion
    if (path := Path(ARGS_COMMAND_PATH)).exists():
        with path.open() as f:
```

**Review by GuyAfik — APPROVED** [review]

File: Tests/demistomock/demistomock.py [hunk 3]
```diff
@@ -455,6 +471,14 @@ def command():
...
 
     """
+    if os.path.exists(ARGS_COMMAND_PATH):
+        with open(ARGS_COMMAND_PATH) as f:
+            try:
+                return json.load(f)["cmd"]
+            except json.JSONDecodeError:
+                return ""
+            except KeyError:
+                return ""
```
Comment (GuyAfik) [inline Tests/demistomock/demistomock.py:481]:
```suggestion
            except (json.JSONDecodeError, KeyError):
                return ""
```

**Review by GuyAfik — APPROVED** [review]

**Comment by ilaner** [comment]
Needs a force merge because pre-commit fails.
The pre-commit thinks the demistomock.py is python 3.10, however it is python 2.7.

**Review by ilaner — APPROVED** [review]

**Pull request merged**